src/signals/incident-management/containers/IncidentSplitContainer/services/useFetchIncident/index.test.js

```diff
@@ -0,0 +1,74 @@
+import { renderHook } from '@testing-library/react-hooks';
+import * as reactRedux from 'react-redux';
+import { showGlobalNotification } from 'containers/App/actions';
+import useFetchIncident from './index';
+
+const dispatch = jest.fn();
+jest.spyOn(reactRedux, 'useDispatch').mockImplementation(() => dispatch);
+
+describe('useFetchIncident service', () => {
+  beforeEach(() => {
+    fetch.resetMocks();
+    dispatch.mockReset();
+  });
+
+  it('should return the empty data when incident no id provided', async () => {
```

Comment:
`asyc` Doesn't seem to be needed; not using `await` in the function body.

Reply:
removed